### PR TITLE
create development guide

### LINKED ISSUE #99: Document how to Build and Deploy
The README must be updated to demonstrate to users how to build and deploy the service.

### PR DESCRIPTION
**Pull request by bjustin-ibm**
Resolves #99

### CODE REVIEW

File: devGuide.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+|Name|Type|Description|
+|---|---|---|
+|INSTANCE|String|A unique identifier for this service. This is useful to differentiate log messages if you run multiple instances of the service|
+|PORT|Integer (default=5000)|The internal port on which the provider REST service should listen|
+|PAYLOAD_LIMIT|Integer (default=900000)|The maxmimum payload size, in bytes, allowed during message batching. This value should be less than your OpenWhisk deployment's payload limit.|
+
+With that in mind, starting the feed service might look something like:
+
+```sh
+docker run KafkaFeedProvider -e DB_URL=https://myDbHost -e DB_USER=MyDbUser -e DB_PASS=MySuperSecret -e PORT=8080 -p 80:8080
```
Comment (bjustin-ibm) [inline devGuide.md]: Thanks! I made both of these changes.

**Comment by bjustin-ibm** [comment]
@dubeejw please review.

**Comment by bjustin-ibm** [comment]
Need to add a mention of the `LOCAL_DEV` environment variable.

**Comment by bjustin-ibm** [comment]
@dubeejw please review

File: README.md [hunk 2]
```diff
@@ -278,8 +278,8 @@ The action caller (you, or your code) must first Base64 encode the data, for exa
 ### Integrating OpenWhisk with IBM Message Hub, Node Red, IBM Watson IoT, IBM Object Storage, and IBM Data Science Experience
 Example that integrates OpenWhisk with IBM Message Hub, Node Red, IBM Watson IoT, IBM Object Storage, IBM Data Science Experience (Spark) service can be [found here](https://medium.com/openwhisk/transit-flexible-pipeline-for-iot-data-with-bluemix-and-openwhisk-4824cf20f1e0).
 
-## Testing
-An OpenWhisk deployment is required in order for the automated test suite to be executed. To run tests locally, run ```$ ./gradlew tests:test -Dhost=<container_address> -Dport=<docker_port>``` from the project's root directory. Where ```<docker_address>``` is the IP address of the service's docker container, and ```<docker_port>``` is the port the docker container is listening on. In addition, ```OPENWHISK_HOME``` must be set to the root of the local OpenWhisk directory. Ex: ```export OPENWHISK_HOME=<openwhisk_directory>```.
+## Development and Testing
+If you wish to deploy the feed service yourself, please refer to the [Development Guide](devGuide.md)
```
Comment (dubeejw) [inline README.md]: Missing period.

Comment (bjustin-ibm) [inline README.md]: @dubeejw fixed.

**Comment by bjustin-ibm** [comment]
@csantanapr please merge.

**Pull request merged**